exercises/querying-workflows/solution/workflow.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+from typing import List
+
+from temporalio import workflow
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+
+@workflow.defn
+class MyWorkflow:
+    def __init__(self) -> None:
+        self._pending_greetings: asyncio.Queue[str] = asyncio.Queue()
+        self._exit = False
+
+    @workflow.run
+    async def run(self) -> List[str]:
+        greetings: List[str] = []
+        workflow.logger.info("Running workflow.")
+        self._current_state = "started"
```

Comment:
While this is technically safe because the query does run after start and it's not used in a signal handler, would recommend this be initialized in the `__init__`